packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -30,6 +30,14 @@ describe('request-queue tests', () => {
     callbacks,
   });
 
+  const getDelayedPromise = () => {
```

Comment:
It does make sense to share this because there are three cases of this within a short space of time.